packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -536,22 +536,6 @@ class FlutterPlugin implements Plugin<Project> {
         if (project.hasProperty('track-widget-creation')) {
             trackWidgetCreationValue = project.property('track-widget-creation').toBoolean()
         }
-        String compilationTraceFilePathValue = null
```

Comment:
These were originally used for the hot code patching feature that has since been removed